io-sim/src/Control/Monad/IOSim/STM.hs

```diff
@@ -261,13 +261,17 @@ data MVarState m a = MVarEmpty   !(Deque (TVar m (Maybe a))) -- blocked on take
 newEmptyMVarDefault :: MonadSTM m => m (MVarDefault m a)
 newEmptyMVarDefault = MVar <$> newTVarIO (MVarEmpty mempty mempty)
 
+labelMVarDefault
+  :: MonadLabelledSTM m
+  => MVarDefault m a -> String -> m ()
+labelMVarDefault (MVar tvar) = atomically . labelTVar tvar . (<> "-MVar")
```

Comment:
For the sake of consistency with other `label*` functions:
```suggestion
labelMVarDefault (MVar tvar) = atomically . labelTVar tvar
```

Reply:
I added this because otherwise it is not possible to tell if a labelled TVar is actually an MVar. But I'm fine with removing it otherwise

Reply:
I see, that's fine.